src/cascadia/LocalTests_TerminalApp/KeyBindingsTests.cpp

```diff
@@ -365,4 +366,94 @@ namespace TerminalAppLocalTests
         }
     }
 
+    void KeyBindingsTests::TestSplitPaneArgs()
+    {
+        // TODO:GH#3536 - These tests _should_ work, but since the LocalTests
+        // fail to run at all right now, I can't be sure that they do. When
+        // #3536 is fixed, make sure that these tests were authored correctly.
+
+        const std::string bindings0String{ R"([
+            { "command": "splitVertical", "keys": ["ctrl+a"] },
+            { "command": "splitHorizontal", "keys": ["ctrl+b"] },
+            { "command": { "action": "splitPane", "split": null }, "keys": ["ctrl+c"] },

```

Comment:
Just curious, what does this do when you have `split = null`?

Reply:
Is this just for warnings?

Reply:
split=null will default to `SplitState::None` which actually does nothing. There's another issues laying around for validating that keybindings have args with actual reasonable values and warning if you don't